src/server/ApiRoutes/Bib.js

```diff
@@ -1,13 +1,9 @@
-import axios from 'axios';
-import appConfig from '../../../appConfig.js';
-
-const appEnvironment = process.env.APP_ENV || 'production';
-const apiBase = appConfig.api[appEnvironment];
+import client from '../routes/client';
 
 function fetchBib(bibId, cb, errorcb) {
-  return axios
-    .get(`${apiBase}/discovery/resources/${bibId}`)
-    .then(response => cb(response.data))
+  return client
```

Comment:
Can we have a more specific naming of this instance? Or any comments to indicate this is from a costumed module to make NYPL internal API call?